packages/dbf/dbf.0.2.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "DBF format parsing"
+description:
+  "This is a small library meant to parse dBase database files. Many cases/variants are not handled but it might already be useful."
+maintainer: ["philippe.veber@gmail.com"]
+authors: ["Léo Andrès" "Philippe Veber"]
+license: "ISC"
+homepage: "https://github.com/pveber/dbf"
+bug-reports: "https://github.com/pveber/dbf/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.6"}
+  "cstruct-unix" {>= "4.0.0"}
+  "ppx_cstruct" {>= "4.0.0"}
```

Comment:
```suggestion
  "ppx_cstruct" {>= "5.2.0"}
```

https://github.com/mirage/ocaml-cstruct/blob/main/CHANGES.md#v520-2020-06-10

for fixing ci lower bounds check errors
```
The following actions will be performed:
=== remove 11 packages
  - remove    base                    v0.14.3           [conflicts with dune]
  - remove    csexp                   1.5.2             [conflicts with dune]
  - remove    dune-configurator       3.11.0            [conflicts with dune]
  - remove    fmt                     0.9.0             [conflicts with ocamlfind]
  - remove    ocaml-compiler-libs     v0.12.4           [conflicts with dune]
  - remove    ocamlbuild              0.14.2
  - remove    parsexp                 v0.14.2           [conflicts with dune]
  - remove    ppxlib                  0.31.0            [conflicts with dune]
  - remove    sexplib0                v0.14.0           [conflicts with dune]
  - remove    stdlib-shims            0.3.0             [conflicts with dune]
  - remove    topkg                   1.0.7             [conflicts with ocamlfind]
=== downgrade 8 packages
  - downgrade cstruct                 6.2.0 to 4.0.0    [required by cstruct-unix]
  - downgrade cstruct-unix            6.2.0 to 4.0.0    [required by dbf]
  - downgrade dune                    3.11.0 to 3.6.0   [required by dbf]
  - downgrade num                     1.4 to 1.0        [required by sexplib]
  - downgrade ocamlfind               1.9.6 to 1.8.0    [required by num]
  - downgrade ppx_cstruct             6.2.0 to 4.0.0    [required by dbf]
  - downgrade ppx_derivers            1.2.1 to 1.0      [upstream or system changes]
  - downgrade sexplib                 v0.14.0 to v0.9.3 [uses jbuilder]
=== recompile 1 package
  - recompile dbf                     0.2.0 (pinned)
=== install 5 packages
  - install   conf-m4                 1                 [required by ocamlfind]
  - install   jbuilder                1.0+beta20.1      [required by ppx_derivers]
  - install   ocaml-migrate-parsetree 1.3.0             [required by ppx_cstruct]
  - install   ppx_tools_versioned     5.2.2             [required by ppx_cstruct]
  - install   result                  1.0               [required by ocaml-migrate-parsetree]
```
```

#=== ERROR while compiling dbf.0.2.0 ==========================================#
# context              2.2.0~alpha3~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/pveber/dbf/releases/download/v0.2.0/dbf-0.2.0.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/dbf.0.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p dbf -j 47 @install
# exit-code            1
# env-file             ~/.opam/log/dbf-7-884080.env
# output-file          ~/.opam/log/dbf-7-884080.out
### output ###
# File "/home/opam/.opam/4.08/lib/sexplib/sexplib.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/sexplib/0/sexplib.0.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/sexplib/num/sexplib.num.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/sexplib/unix/sexplib.unix.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ppx_derivers/ppx_derivers.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "lib/dune", line 5, characters 14-31:
# 5 |   (preprocess (pps ppx_cstruct)))
#                   ^^^^^^^^^^^^^^^^^
# (cd _build/.sandbox/82a9cad81695bee4217f9325d759693f/default && .ppx/dddc94fd80d07bf5980184f4fc0c28b4/ppx.exe --cookie 'library-name="dbf"' -o lib/dbf.pp.ml --impl lib/dbf.ml --dump-ast)
# Fatal error: exception File "lib/dbf.ml", line 135, characters 2-6: let operators are not supported before OCaml 4.08
```